README.md

```diff
@@ -14,8 +14,8 @@ Streamlit lets you turn data scripts into sharable web apps in minutes, not week
 pip install streamlit
 streamlit hello
 ```
+Streamlit can also be installed in the virtual environment of [Windows](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-windows), [Mac](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux), and [Linux](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux).
```

Comment:
"Streamlit can also be installed in a virtual environment on"

This is important since otherwise it sounds like you need a Windows VM to install streamlit (as opposed to a VM on Windows)

Reply:
It is like "Can also be installed"

Reply:
If you have any suggestions then let me know.